### ISSUE TITLE
Enums missing from System.Security.Cryptography.X509Certificates.

### ISSUE THREAD

**Issue by vladimir-kazakov** [dotnet/corefx#17194]
The following enums are missing from the System.Security.Cryptography.X509Certificates namespace:

* [TrustStatus](https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.truststatus(v=vs.110).aspx).
* [X509SelectionFlag](https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.x509selectionflag(v=vs.110).aspx).

Both are available in .NET Framework.

I could try to add them myself, but I have no experience contributing to .NET Core. Perhaps someone from the owners of this project could clarify a few things:

1. First of all, are these enums really needed in .NET Core? As far as I understand, the goal is to bring all APIs of .NET Framework to .NET Standard 2.0, which means that these enums are needed, right?
2. Do I understand correctly that it's just a matter of adding these enums to the System.Security.Cryptography.X509Certificates project? Or maybe something additionally has to be done? Perhaps /ref/System.Security.Cryptography.X509Certificates.cs has to be changed as well?

**Comment by bartonjs** [comment]
@vladimir-kazakov Is there a particular scenario that these enums are needed for? They are only used by types that themselves aren't part of .NET Core (or .NET Standard), so it doesn't seem to make a whole lot of sense to have them just as exported values.

**Comment by vladimir-kazakov** [comment]
Sorry, I was wrong.

I thought that .NET Standard provides all APIs of a specific version of .NET Framework (or at least that's the goal). The announcement of .NET Standard 2.0 confused me even more by saying that a lot of .NET Framework 4.6.1 APIs will be added to this version of .NET Standard. So, I thought that the mentioned types are missing, since they're a part of .NET Framework 4.6.1.

I reviewed this topic again, and now I see that .NET Standard is just a subset of APIs of a specific version of .NET Framework. In other words, if .NET Framework 4.6.1 implements .NET Standard 2.0, this doesn't mean that .NET Standard 2.0 has all APIs of .NET Framework 4.6.1. According to .NET Standard docs (https://github.com/dotnet/standard/tree/master/docs/versions), the types that I thought are missing are not a part of any version of .NET Standard, which explains why they don't exist in .NET Core.

This issue is not an issue. Closing it.
